src/charm.py

```diff
@@ -169,6 +171,25 @@ def _with_console_address(self, minio_args):
         console_port = str(self.model.config["console-port"])
         return [*minio_args, "--console-address", ":" + console_port]
 
+    def _configure_ingress(self, interfaces):
+        if interfaces["ingress"]:
+            interfaces["ingress"].send_data(
+                {
+                    "prefix": "/",
```

Comment:
This'll define the endpoint that we claim on the ingress, as written we've claimed the root (or really, `http://INGRESS_URL/`).  It probably makes the most sense that this be configurable via a config option.  The config option could accept anything that would be valid in a URL (a good default value could be `/minio`).